pkgs/applications/networking/browsers/qutebrowser/default.nix

```diff
@@ -1,25 +1,27 @@
-{ stdenv, fetchurl, python, buildPythonPackage, qt5, pyqt5, jinja2, pygments, pyyaml, pypeg2}:
+{ stdenv, fetchgit, python, buildPythonPackage, qt5, pyqt5, jinja2, pygments, pyyaml, pypeg2 }:
 
-let version = "0.2.1"; in
+let version = "0.3-pre"; in
```

Comment:
Better change to the output of `git describe` 
If available. Or change `pre` to the date of commit.